components/eamxx/docs/user/aerocom_cldtop.md

```diff
@@ -0,0 +1,30 @@
+# The AeroCom algorithm
+
```

Comment:
My vision is for the user guide to explain available options to control a simulation without changing code, the dev guide should provide general info about how to change code, and the tech doc should describe the equations solved and algorithms used for the particular version of EAMxx we have now. 

In this context, I think aerocom_top.md is a tech doc and clean_clear_sky.md is mostly a user guide thing except for describing the addition of a new class which seems like it will be outdated soon (when the new class isn't new) and maybe doesn't need to be described?

Reply:
I think we're in agreement now. I'm only complaining about mentioning classes because I think 99% of users won't understand what a class is (and I think you've deleted the class mention now). I totally agree with "I think it is important people know what we are calling "clean" and "clean-clear" at a brief technical level"